contentcuration/contentcuration/frontend/channelList/views/KCard/index.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+
+  <BaseCard
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+  >
+    <div>
+      <KImg
+        v-if="layout === 'vertical' && thumbnailDisplay !== 'none'"
+        :src="thumbnailSrc"
+        :height="300"
+        :width="500"
+        :isDecorative="false"
+        altText="thumbnail image"
+        :appearanceOverrides="{
+          objectFit: thumbnailScaleType,
+          backgroundColor: $themeTokens.fineLine
```

Comment:
I don't see anything in the design that would indicate we need to be overriding background color of `KImg`'s letterbox envelope. What was the reason for this?